test/resources/Autofill.test.ts

```diff
@@ -0,0 +1,53 @@
+import nock from "nock";
+import { Onfido } from "onfido-node";
+
+const onfido = new Onfido({ apiToken: "api_token" });
+
+const exampleAutofillJson = {
+  document_id: "21345-xxx",
+  document_classification: {
+    issuing_country: "FRA",
+    document_type: "national_identity_card"
+  },
+  extracted_data: {
+    date_of_birth: "1990-07-21",
+    date_of_expiry: "2025-07-07",
+    document_number: "200407512345",
+    first_name: "AMANDINE CHANTAL",
+    gender: "Female",
+    last_name: "MAVARINE",
+    mrz_line1: "IDFRAMAVARINE<<<<<<<<<<<<<<<<<075123",
+    mrz_line2: "2004075123457AMANDINE<CHANT9007219F5",
+    nationality: "FRA"
+  }
+};
```

Comment:
I'm sure it is, but just checking for peace of mind - this is sample data right?

Reply:
It's copied from the docs (https://documentation.onfido.com/?http#extraction-result) but maybe it should be made more obviously fake for tests

Reply:
Good point, as Ciaran have answered - it is the test data from documentation. I've added comment in the tests.